src/services/search/model/SearchResponse.res

```diff
@@ -0,0 +1,34 @@
+type image = {
```

Comment:
These types are very similar to the types used for the UI, maybe we can unify them?

Reply:
you can use them in the UI but I need to model the response of Spotify. That's the thing with typed languages but it is also good practice to have two similar models for this case